src/components/dropdown/dropdown.js

```diff
@@ -100,7 +108,7 @@ export default class Dropdown extends PureComponent {
       up,
       hoverable,
       label,
-      onChange,
+      onChange: omit, // not used in this scope
```

Comment:
`onChange` is not used in the render method and deleting it here changes the snapshots, consider removing it in future releases.